assets/component-newsletter.css

```diff
@@ -50,6 +50,12 @@
   height: 100%;
 }
 
+.newsletter-form__button:focus-visible,
+.newsletter-form__button:focus {
```

Comment:
One thing, and I haven't tested it here but those two selectors need to be separated as Safari won't read the styling for `:focus` because it doesn't support `:focus-visible`.

Reply:
This has been updated, but I noticed the focus doesnt seem to work on main for the newsletter button either 🤔 https://github.com/Shopify/dawn/pull/1097/commits/23d48c6a5a5404efc097234ff61509bc9c746227